src/alvarium/annotators/handler/contracts.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from enum import Enum
+
+HTTP_REQUEST_KEY = "HttpRequestKey"
+CONTENT_TYPE = "Content-Type"
+CONTENT_LENGTH = "Content-Length"
```

Comment:
I'm not a fan of global constants in python. It opposes a risk of variable reassignment as python does not provide a way to create const variables. You have three solutions:
- Either encapsulate these variables in a class and expose them through getters
-  or create an `Enum` with these values.
- or use the `Constants` meta class, see [here](https://stackoverflow.com/a/19306516)